solr/solrj/src/java/org/apache/solr/client/solrj/impl/CloudHttp2SolrClient.java

```diff
@@ -147,6 +147,7 @@ public static class Builder {
     private ResponseParser responseParser;
     private long retryExpiryTime =
         TimeUnit.NANOSECONDS.convert(3, TimeUnit.SECONDS); // 3 seconds or 3 million nanos
+    private int timeToLive = 60;
```

Comment:
The name should reflect the units in a suffix

Reply:
Done....     I also renamed a related timeToLive that is in milliseconds to `timeToLiveMs`...